warehouse/core/src/main/java/datawave/attribute/EventField.java

```diff
@@ -0,0 +1,41 @@
+package datawave.attribute;
+
+import datawave.util.StringUtils;
+
+/**
+ * FIELD_NAME.COMMONALITY.GROUPING_CONTEXT
+ */
+public class EventField {
+    
+    public static String[] getCommonalityAndGroupingContext(String field) {
+        String[] splits = StringUtils.split(field, '.');
```

Comment:
use GROUPING_SEPARATOR_CHARACTER from JexlASTHelper for '.' maybe

Reply:
Otherwise a static final in this class

Reply:
Sure